runtime/v1/linux/proc/process.go

```diff
@@ -23,7 +23,11 @@ import (
 )
 
 // RuncRoot is the path to the root runc state directory
-const RuncRoot = "/run/containerd/runc"
+const (
+	RuncRoot = "/run/containerd/runc"
+	// StoppedPID is the pid assigned after a container has run and stopped
+	StoppedPID = -1
```

Comment:
This is a breaking change yea? `Pid` used to remain the pid even after stop right?

Reply:
i don't think it's a breaking change  because a pid that doesn't exist is just incorrect.  It's better to return 0 or -1 in these cases when the container is stopped then someone trying to signal a pid that could be reused by someone else.